projects/solar/inc/relay_fsm.h

```diff
@@ -8,14 +8,23 @@
 #include "drv120_relay.h"
 #include "event_queue.h"
 #include "fsm.h"
+#include "soft_timer.h"
+#include "solar_boards.h"
 #include "status.h"
 
+#define CURRENT_ASSERT_THRESHOLD_uA 10
+
+#define RELAY_ASSERTION_DELAY_MS 10
+#define DATA_STORE_ASSERTION_DELAY_MS 1000
```

Comment:
You should add comments saying what each of these is (e.g. the current assert threshold one is the current threshold below which we consider the relay disconnected).

Reply:
done